### DISCUSSION TITLE
Pregabalin Seizure Concerns

### DISCUSSION THREAD

**Discussion by utaninja** [TripSit/drugs#40] [Pharmacology]
This is a discussion made to communicate discussion over Pregabalin's proposed seizure concerns.

Through various online communities, there has been a lot of talk of Pregabalin and its seizure concerns.
I have heard dozens of anecdotal reports directly from individuals reporting experiencing seizures while under the influence of Pregabalin (generally in a recreational setting).

To note- I have reached out to the moderators at r/gabagoodness but have yet to hear back. The staff there have continuously claimed that pass 600mg is the point of concern for seizures, from my understanding of what they are saying. I may not be citing them properly, you can see these messages in the Reddit posts linked below.  I have yet to find a basis for this, and it seems that people have reported seizures at low dosages aswell.
It seems that they have also indicated multiple different types of seizures are at risk while on Pregabalin.

Just to cite some example posts online about it's seizure risks-
https://www.reddit.com/r/gabagoodness/comments/111hnbq/had_a_seizure_from_600mg_lyrica_300mg_redose_will/
https://www.reddit.com/r/gabagoodness/comments/z45ypv/i_overdosed_on_pregabalin_and_had_2_really_bad/
https://www.reddit.com/r/gabagoodness/comments/10yos15/how_does_a_seizure_on_pregabalin_looks_like/
https://www.reddit.com/r/gabagoodness/comments/w832w5/what_kind_of_seizures_does_pregabalin_cause/
https://www.reddit.com/r/gabagoodness/comments/115tz8s/pregabalintwitching/
https://bluelight.org/xf/threads/seizures-from-pregabalin-lyrica.896412/

I have come across this case report as well https://www.tandfonline.com/doi/full/10.1080/24734306.2018.1458465#:~:text=Toxicity%20may%20occur%20after%20overdose,seizure%20with%20confirmatory%20serum%20concentrations.
I have continued the author of this case study in hopes of more insight and information.

This is an aggregation of what I know of this topic. My knowledge of pharmacology is very limited and to be honest I don't fully understand what is happening here. I would appreciate any insight, discussion, or suggestions to further help this.

**Comment by zelixir25** [comment]
is the purpose of determining the seizure risk so that we can better determine theoretical risk when combined with other drugs that can lower seizure threshold?

**Reply by utaninja** [reply]
Yes, and if certain drugs can increase the risk of seizures but not necessarily via methods of lowering the seizure threshold chemically speaking. (as in psychological reasons from my understanding)

**Comment by utaninja** [comment]
Looks like I found a source that says Pregabalin is proconvulsant!! https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7688538/

**Comment by utaninja** [comment]
Forwarding discussion with @zelixir25 

(uta)
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4526718/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4492734/

(uta) 
Seems this should be enough to show pregabalin's seizure risks (on top of the other studies collected) but if there a risk of seizures due to this, would that then mean when someone takes a drug with a seizure risk (say tramadol) would that combine and be a total greater risk overall?

As in- do the fact that both of them have risks inherently raise the risks for both, or does it not effect it and therefore less of a interaction concern but simply the concern is only for that drug itself. - Pregabalin has seizure risks, you should do XYZ because of this, but taking other things with seizure risks effect this. (or what I would suspect on intuition, the opposite)

(bread)
usually taking 2 drugs that lower seizure threshold is more dangerous than just taking 1

(uta)
So seizure threshold is more of a blanket thing?
as in- not different types depending on the cause that are seperate from one another
these things all just contribute to lowering seizure threshold (etc)

We would like to verify this with someone with a neuropharm background and will move forward with Pregabalin's interactions if this is the case!

**Comment by utaninja** [comment]
To sum up the question-

We are investigating Pregabalin's possible seizure concerns. How it may work, and it's interaction with other drugs and how they may effect this concern. An example of this concern may be for Pregabalin and Tramadol, Tramadol being known to lower seizure thresholds and cause seizures. Another example would be amphetamine, does amphetamine when taken under the influence of Pregabalin increase the possible seizure concern of amphetamine? This is all focused on recreational dosages of Pregabalin (900-1200mg+), though information on all dosage ranges would be helpful.

I've come across these two studies https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4526718/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4492734/ which would seem to indicate a mechanism for Pregabalin causing seizures. If this is the case, would that then mean when someone takes a drug with a seizure risk (say tramadol) would that combine and be a total greater risk overall? As in, you have X risk due to Tramadol, taking Pregabalin increases X+Y to a total of a Z amount. Or, is it that you have X risk via a certain manner, and Y risk, and they don't effect each other?

I've come across the following information that may be helpful-
Slocum, G. W., Schult, R. F., Gorodetsky, R. M., Wiegand, T. J., Kamali, M., & Acquisto, N. M. (2018). Pregabalin and paradoxical reaction of seizures in a large overdose. Toxicology Communications, 2(1), 19–20. https://doi.org/10.1080/24734306.2018.1458465
Isoardi, K. Z., Polkinghorne, G., Harris, K., & Isbister, G. K. (2020). Pregabalin poisoning and rising recreational use: a retrospective observational series. British journal of clinical pharmacology, 86(12), 2435–2440. https://doi.org/10.1111/bcp.14348

I've also collected a series of online posts about it, either people reporting seizures they experienced under the influence of Pregabalin or discussions about it. You can see this here: https://pastebin.com/1RUrXVN3

Thank you!

**Comment by utaninja** [comment]
I had asked Becks from Bluelight for some help on this question. With this information and more that I've looked into, I feel comfortable going through with Pregabalin's interactions in regards to all of it's seizure concerns. Here is a forwarding of what Becks sent-

__[Drugs that lower the seizure threshold - Hitchings](https://openaccess.sgul.ac.uk/id/eprint/108264/1/Hitchings%20-%20Drugs%20that%20lower%20the%20seizure%20threshold%20-%20Author%20final%20version.pdf)__ 
*by a senior lecturer in clinical pharmacology and honorary consultant in neuro-intensive care*
## Seizures and the seizure threshold
### Terminology 
- Seizure threshold: minimum intensity of a stimulus required to induce a seizure. Primarily an experimental phenomenon.
### Pathophysiology and pharmacoloy
- Seizures happen when there is an excess of excitatory activity (glutamate) relative to inhibitory activity (GABA). 
 - Glutamate mechanisms: NMDA, AMPA, kainite receptors -> influx of sodium/calcium ions -> depolarization favored
 - GABA mechanisms: influx of chloride ions
- The propensity of a drug to induce seizures depends on its effects on neurotransmission and their timecourse (e.g. whether it increases seizure risk during use or on withdrawal), the concentration of drug reaching the brain, and the susceptibility of the individual patient.
 - Generally, drugs that contribute to this balance of excitatory/inhibitory signals lead to a lower seizure threshold (this is more helpful to say than "incite seizures").
 ## Specific drugs
### Analgesics
**Opioid analgesics:**
- Depends on the drug (diverse effects on the seizure threshold - e.g., morphine potentiates chemically-induced seizures but protects against electrically-induced seizures) and the dosage (e.g., morphine is anticonvulsant in low doses but pro-convulsant at high doses - this pattern is reversed for fentanyl). 
- **Tramadol** - commonly associated with seizures in *overdose*.
 - Its seizure-inducing effects are not likely mediated through serotonin. 
 - They may be due to opioid-dependent inhibition of GABAergic pathways or histamine H1 receptor activation. 
### Antipsychotics
- Clozapine has the most marked effects on seizure threshold, followed by other second gen antipsychotics (except for aripiprazole/Abilify, which may have anticonvulsant properties - this means during WD threshold may be lowered).
- First gen APs only have a modest effect.
### Antidepressants 
- Most antidepressants (SSRIs, SNRIs, monoamine oxidase A inhibitors, and mirtazapine) are anticonvulsant at therapeutic doses. TCAs are mostly an exception to this. 
- During **overdose**, seizures are common (especially with TCAs): this is likely due to inhibition of GABAergic and histaminergic transmission.
- **Bupropion/Wellbutrin** has a dose-dependent relationship with seizure risk and may be important for individuals with other susceptibility factors. 
### Antiepileptics
- Generally reduce seizure threshold, except in toxic conencrations (due to chronic overuse in the case of benzos/barbiturates, overdose in the case of gabapentin or carbamazepine/lamotrigine, or altered pharmacokinetics due to combinations with other drugs) or withdrawals.
### Anaesthetics 
- Lidocaine effects on seizure threshold vary with concentration in a bimodal fashion. 
 - Low conc: supression
 - As concentration increases, it lowers seizure threshold until eventually it induces seizures (in most subjects) 
 - *May be important to mention this for entry for cocaine*
 
From that paper, it looks like anything that disrupts excitatory/inhibitory balance in the direction of excitation will increase likelihood of seizures
In other words, things that lower seizure threshold in that way are additive in effects
Tramadol overdose specifically is what needs to be worried about in this case
Same with pregablin overdose
So if someone were to take high doses of both this would be a risk
However at therapeutic doses of both it appears to be safe with regards to seizures
Generally if someone is planning on combining them, lower doses than normal of both should be taken to avoid excess respiratory depression due to synergy
This is also the case for gabapentin
I hope this helps 🙂

**Reply by utaninja** [reply]
I had two following questions to start with-
Would this include something like amphetamines (due to how amphetamine works)? So high dose amphetamines and pregabalin for example should be avoided. But medical dosages should be fine.

Do you know if a susceptibility factor may include something like jogging?

Becks response:
Yes, stimulants like coke and amph (especially at high doses) lower seizure threshold. I think something that definitely should be specifically mentioned is how lidocaine can SIGNIFICANTLY lower threshold and it is a common cut in coke
Susceptibility factors can include low sodium (so if someone hasn't been eating for a while), high fever/illness, lack of sleep... I've even done a bit of research and hot water sensations on the top of your head can trigger a seizure in some people (so showers are a risk). Also obv GABA wd

- We then talked a bit but concluded it seems that-
Pregabalin has A seizure risk. In the case physical activity (B) is a susceptibility factor, than it would be
A + B +  C = Z
C being all other factors that may add to the seizure except physical activity
Z being total seizure risk for the individual

It may be additive or a multiplier depending on the interaction/susceptibility. Thanks Becks!

**Comment by utaninja** [comment]
(rough draft)
This is a baseline set of studies that I believe would be good to add to Pregabalin's interactions when relevant to seizure discussions and Pregabalin in general. I would also like to note it is a anti convulsant at lower dosages. For interactions, we aim to display in the highest concern, in this case Pregabalin at recreational dosages (600mg+). It is also good to mention in the interaction note if the interaction is less/not a concern when lower dosages are used.

---
Studies:
  Barker-Haliski, M., & White, H. S. (2015). Glutamatergic Mechanisms Associated with Seizures and Epilepsy. Cold Spring Harbor Perspectives in Medicine, 5(8), a022863. https://doi.org/10.1101/cshperspect.a022863
  
  Ben-Menachem, E. (2004). Pregabalin pharmacology and its relevance to clinical practice. Epilepsia, 45 Suppl 6, 13–18. https://doi.org/10.1111/j.0013-9580.2004.455003.x
  
  French, J., Kwan, P., Fakhoury, T., Pitman, V., DuBrava, S., Knapp, L., & Yurkewicz, L. (2014). Pregabalin monotherapy in patients with partial-onset seizures. Neurology, 82(7), 590–597. https://doi.org/10.1212/WNL.0000000000000119
  
  Gayatri, N. A., & Livingston, J. H. (2006). Aggravation of epilepsy by anti-epileptic drugs. Developmental Medicine and Child Neurology, 48(5), 394–398. https://doi.org/10.1017/S0012162206000843
  
  Hitchings, A. W. (2016). Drugs that lower the seizure threshold. Adverse Drug Reaction Bulletin, 298(1), 1151–1154. https://doi.org/10.1097/FAD.0000000000000016
  
  Isoardi, K. Z., Polkinghorne, G., Harris, K., & Isbister, G. K. (2020). Pregabalin poisoning and rising recreational use: a retrospective observational series. British Journal of Clinical Pharmacology, 86(12), 2435–2440. https://doi.org/10.1111/bcp.14348
  
  Kumar, N., Laferriere, A., Yu, J. S. C., Leavitt, A., & Coderre, T. J. (2010). Evidence that pregabalin reduces neuropathic pain by inhibiting the spinal release of glutamate. Journal of Neurochemistry, 113(2), 552–561. https://doi.org/10.1111/j.1471-4159.2010.06625.x
  
  Slocum, G. W., Schult, R. F., Gorodetsky, R. M., Wiegand, T. J., Kamali, M., & Acquisto, N. M. (2018). Pregabalin and paradoxical reaction of seizures in a large overdose. Toxicology Communications, 2(1), 19–20. https://doi.org/10.1080/24734306.2018.1458465
  
---
Reports:
https://www.reddit.com/r/gabagoodness/comments/111hnbq/had_a_seizure_from_600mg_lyrica_300mg_redose_will/
https://www.reddit.com/r/gabagoodness/comments/z45ypv/i_overdosed_on_pregabalin_and_had_2_really_bad/
https://www.reddit.com/r/gabagoodness/comments/10yos15/how_does_a_seizure_on_pregabalin_looks_like/
https://www.reddit.com/r/gabagoodness/comments/w832w5/what_kind_of_seizures_does_pregabalin_cause/
https://www.reddit.com/r/gabagoodness/comments/115tz8s/pregabalintwitching/
https://www.bluelight.org/community/threads/seizures-from-pregabalin-lyrica.896412/


  Altair. (n.d.-a). Pregabalin (Lyrica) - Erowid Exp - “Heart Arrhythmia, Seizure : Not Recreational.” Retrieved May 21, 2024, from https://www.erowid.org/experiences/exp.php?ID=69917
  Douke, R. (n.d.). Pregabalin & Tramadol - Erowid Exp - “Grand Mal Seizures.” Retrieved May 21, 2024, from https://www.erowid.org/experiences/exp.php?ID=83232
  Sean. (n.d.-b). Pregabalin - Erowid Exp - “Stay Away.” Retrieved May 21, 2024, from https://www.erowid.org/experiences/exp.php?ID=67302
  
  -
Pregabalin has been shown to induce and or increase the risk of seizures at recreational dosages. There is little evidence showing there is a concern at medical dosages (600mg and under). This risk may be increased when mixed with other drugs that lower seizure threshold. Caution is recommended when using doses of higher than 600mg. Reducing various susceptibility factors may also be beneficial in lowering this risk. Susceptibility factors may include things such as low sodium, lack of sleep, and intensive physical activity.

**Reply by utaninja** [reply]
Revision + source addition

Pregabalin has been shown to induce and or increase the risk of seizures at recreational dosages. There is little evidence showing there is a concern at medical dosages (600mg and under). This risk may be increased when mixed with other drugs that lower seizure threshold. Caution is recommended when using doses of higher than 600mg. Reducing various susceptibility factors may also be beneficial in lowering this risk. Susceptibility factors may include things such as low sodium, lack of sleep, and intensive physical activity.

  Altair. (n.d.-a). Pregabalin (Lyrica) - Erowid Exp - “Heart Arrhythmia, Seizure : Not Recreational.” Retrieved May 21, 2024, from https://www.erowid.org/experiences/exp.php?ID=69917
  Barker-Haliski, M., & White, H. S. (2015). Glutamatergic Mechanisms Associated with Seizures and Epilepsy. Cold Spring Harbor Perspectives in Medicine, 5(8), a022863. https://doi.org/10.1101/cshperspect.a022863
  Ben-Menachem, E. (2004). Pregabalin pharmacology and its relevance to clinical practice. Epilepsia, 45 Suppl 6, 13–18. https://doi.org/10.1111/j.0013-9580.2004.455003.x
  Douke, R. (n.d.). Pregabalin & Tramadol - Erowid Exp - “Grand Mal Seizures.” Retrieved May 21, 2024, from https://www.erowid.org/experiences/exp.php?ID=83232
  French, J., Kwan, P., Fakhoury, T., Pitman, V., DuBrava, S., Knapp, L., & Yurkewicz, L. (2014). Pregabalin monotherapy in patients with partial-onset seizures. Neurology, 82(7), 590–597. https://doi.org/10.1212/WNL.0000000000000119
  Gayatri, N. A., & Livingston, J. H. (2006). Aggravation of epilepsy by anti-epileptic drugs. Developmental Medicine and Child Neurology, 48(5), 394–398. https://doi.org/10.1017/S0012162206000843
  Hitchings, A. W. (2016). Drugs that lower the seizure threshold. Adverse Drug Reaction Bulletin, 298(1), 1151–1154. https://doi.org/10.1097/FAD.0000000000000016
  Kumar, N., Laferriere, A., Yu, J. S. C., Leavitt, A., & Coderre, T. J. (2010). Evidence that pregabalin reduces neuropathic pain by inhibiting the spinal release of glutamate. Journal of Neurochemistry, 113(2), 552–561. https://doi.org/10.1111/j.1471-4159.2010.06625.x
  Oskouei, Z., Moshiri, M., Raouf-Rahmati, A., Nemati, A., Bemani Naeini, M., Jomehpour, H., Roohbakhsh, A., Salmasi, Z., & Etemad, L. (2025). Seizures as an Adverse Effect of Pregabalin Consumption: A Systematic Review. Addiction & Health, 17, 1527. https://doi.org/10.34172/ahj.1527
  Rietjens, S. J., Sikma, M. A., Hunault, C. C., De Lange, D. W., & Hondebrink, L. (2022). Pregabalin poisoning: Evaluation of dose‐toxicity relationship. British Journal of Clinical Pharmacology, 88(3), 1288–1297. https://doi.org/10.1111/bcp.15073
  Sean. (n.d.-b). Pregabalin - Erowid Exp - “Stay Away.” Retrieved May 21, 2024, from https://www.erowid.org/experiences/exp.php?ID=67302
  Slocum, G. W., Schult, R. F., Gorodetsky, R. M., Wiegand, T. J., Kamali, M., & Acquisto, N. M. (2018). Pregabalin and paradoxical reaction of seizures in a large overdose. Toxicology Communications, 2(1), 19–20. https://doi.org/10.1080/24734306.2018.1458465
